modules/persistent-storage-csi-cinder-storage-class.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-cinder.adoc
+
+[id="persistent-storage-csi-cinder_{context}"]
+= Making OpenStack Cinder CSI the default storage class
+
+In {product-title}, the default storage class references the in-tree Cinder driver. The storage class will default to referencing OpenStack Cinder CSI in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree storage class are planned for migration to the OpenStack Cinder CSI storage class at that time.
+
+The OpenStack Cinder CSI driver uses the `cinder.csi.openstack.org` parameter key to support dynamic provisioning.
+
+.Procedure
+To enable the use of OpenStack Cinder CSI in {product-title} {product-version}, you must request the `standard-csi` storage class to overwrite the default in-tree storage class.
+
+. Create the `PersistentVolumeClaim` object using the Cinder CSI storage class, as shown in the following example:
++
+[source,yaml]
+--
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cinder-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: standard-csi <1>
+--
+<1> Assigns the default storage class.
```

Comment:
If we chose to document how to create a PVC that is using the `standard-csi` storage class then  `default` is extra here.